proto/cosmos/gov/v1beta2/genesis.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package cosmos.gov.v1beta2;
+
+import "gogoproto/gogo.proto";
+import "cosmos/gov/v1beta2/gov.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types";
```

Comment:
Is the pattern that the latest version is just used as `/types` with no version in the package?

Reply:
This is what I've defaulted to doing but also a question I had myself

Reply:
Well I don't actually think we have a good pattern right now. I proposed a pattern here which just uses the proto package as the go package more or less: https://github.com/cosmos/cosmos-sdk/discussions/10582. But that's a bigger refactoring. And for now this seems the most logical and least intrusive. But open to other suggestions